app/src/main/java/org/sopt/sample/presentation/gallery/GalleryFragment.kt

```diff
@@ -0,0 +1,44 @@
+package org.sopt.sample.presentation.gallery
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import org.sopt.sample.R
+import org.sopt.sample.base.BindingFragment
+import org.sopt.sample.databinding.FragmentGalleryBinding
+import org.sopt.sample.presentation.gallery.data.GalleryRecycleData
+import org.sopt.sample.presentation.gallery.diffUtilAdapter.GalleryListAdapter
+import org.sopt.sample.presentation.home.HomeFragment
+
+
+class GalleryFragment() : BindingFragment<FragmentGalleryBinding>(FragmentGalleryBinding::bind,R.layout.fragment_gallery) {
+
+    private val dataList = listOf<GalleryRecycleData>(
```

Comment:
```suggestion
    private val dataList = listOf(
```
담으신 마블 데이터(GalleryRecycleData)로 타입추론이 가능하기 때문에 생략도 가능합니당